src/common/src/FileUpload/FileUpload.tsx

```diff
@@ -80,9 +82,15 @@ const FileUpload: React.FC<Props> = ({
 					)}
 				</FileUploadContentStyle>
 			</FileUploadStyle>
-			{files?.length > 0 && (
+			{(files?.length > 0 || (storedFiles && storedFiles.length > 0)) && (
 				<FileList
-					files={removeCompletedFromList ? files.filter((file) => file.progress !== 100) : files}
+					files={
+						storedFiles
```

Comment:
I guess if you pass the prop once you are stuck with managing the list in the containing component? Did you consider also allowing for a default value approach?